src/solution.jl

```diff
@@ -0,0 +1,57 @@
+abstract type AbstractSolution end
+
+"""
+    feasibility_cut(sol::AbstractSolution)
+
+Returns the tuple `(a, α)` representing the feasibility cut ``⟨a, x⟩ ≧ α`` certified by this solution.
+"""
+function feasibility_cut end
```

Comment:
Minor point, but none of the other methods have `_` so we should just be consistent with `feasibilitycut`.

Reply:
I am ok with renaming it `feasibilitycut` :)